pkg/apis/system/v1beta1/types.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1beta1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+const (
+	BundleStatusNone       = "None"
+	BundleStatusCollecting = "Collecting"
+	BundleStatusCollected  = "Collected"
```

Comment:
Should there be BundleStatus type to follow the convention?

Reply:
Updated..